pinot-broker/src/main/java/org/apache/pinot/broker/broker/helix/BaseBrokerStarter.java

```diff
@@ -392,6 +394,11 @@ public void start()
             timeSeriesRequestHandler, _responseStore);
     _brokerRequestHandler.start();
 
+
+    LOGGER.info("Initializing BrokerGrpcServer");
+    _brokerGrpcServer = new BrokerGrpcServer(_brokerConf, brokerId, _brokerMetrics, _brokerRequestHandler);
+    _brokerGrpcServer.start();
```

Comment:
Move this after `Starting broker admin application` to ensure query related things are setup (e.g. CPU time measurement)

Reply:
done